src/models/message.ts

```diff
@@ -0,0 +1,40 @@
+import { Guid } from "guid-typescript";
+
+/**
+ * Message model.
+ */
```

Comment:
Useless comment, the name Message inside the package models already implies that this is a model for Messages.

Reply:
Added for consistency, for when generating TSDoc for example. This is also done in for example blocker-models.